packages/neos-ui-editors/src/Editors/Image/index.js

```diff
@@ -215,8 +215,17 @@ export default class ImageEditor extends Component {
     }
 
     handleChooseFile = () => {
-        this.previewScreen.chooseFromLocalFileSystem();
-        this.setState({isAssetLoading: true});
+        const {secondaryEditorsRegistry, options} = this.props;
+        const {component: MediaUploadScreen} = secondaryEditorsRegistry.get('Neos.Neos/Inspector/Secondary/Editors/MediaUploadScreen');
```

Comment:
IMO we should use the wording `AssetUploadScreen` here as all our models are called something `Asset...`

Reply:
Thank you for the fast response. I adjusted the naming.